app/controllers/subgroups_controller.rb

```diff
@@ -78,28 +144,29 @@ def organizer_params
               :password,
               phone_numbers_attributes: [:number],
               email_addresses_attributes: [:address],
-              personal_addresses_attributes: [:postal_code])
+              personal_addresses_attributes: [:postal_code]
+             )
        .tap { |prams| format(prams) }
   end
 
-  def format(prams)
-    prams
+  def format(params)
+    params
```

Comment:
hmm... i'd actually prefer `prams` here. `params` shadows a variable that is already in scope for the controller. which is why i used `prams`. fine with some other convention, but whatever we call it, i'd offer that it introduces unhelpful confusion to call it `params`

Reply:
(could call it `_params` if the mis-spelling also introduces unnecessary confusion?)